tests/jax/components/building/parameter_client_test.py

```diff
@@ -287,7 +287,7 @@ def test_executor_parameter_client_evaluator_with_parameter_client(
     mock_builder = mock_builder_with_parameter_client
     mock_builder.store.is_evaluator = True
     exec_param_client = ExecutorParameterClient(
-        config=ExecutorParameterClientConfig(executor_parameter_update_period=100)
+        config=ExecutorParameterClientConfig(executor_parameter_update_period=200)
```

Comment:
not imp but why was this increased?

Reply:
This was increased to match the assertion test: `assert mock_builder.store.executor_parameter_client._update_period == 200`